common/log/tag/tags.go

```diff
@@ -51,6 +51,14 @@ func Timestamp(timestamp time.Time) Tag {
 	return newTimeTag("timestamp", timestamp)
 }
 
+func EarliestTime(time int64) Tag {
```

Comment:
if I may suggest making it `*int65`
```suggestion
func EarliestTime(time *int64) Tag {
  if time == nil {
    return newInt64("earliest-time", -1) 
  }
  return newInt64("earliest-time", time)
}
```